packages/mds-stream/index.ts

```diff
@@ -135,15 +167,25 @@ async function writeStreamBatch(stream: Stream, field: string, values: unknown[]
 
 // put basics of vehicle in the cache
 async function writeDevice(device: Device) {
+  if (env.SINK) {
+    return write('mds.device', JSON.stringify(device))
+  }
   return writeStream(DEVICE_INDEX_STREAM, 'data', device)
 }
 
 async function writeEvent(event: VehicleEvent) {
+  if (env.SINK) {
+    return write('mds.event', JSON.stringify(event))
+  }
   return Promise.all([DEVICE_RAW_STREAM, PROVIDER_EVENT_STREAM].map(stream => writeStream(stream, 'event', event)))
```

Comment:
Notice that events get written to 2 different Redis streams, but only one knative. I am not sure how the `provider:event` stuff translates to knative.